draft-ietf-ppm-dap.md

```diff
@@ -2436,195 +2436,55 @@ suite:
-     * Applying differential privacy to either measurements before sharding them
-       into reports or to aggregate shares ({{dp}}) can protect isolated
-       legitimate measurements.
-
-### Aggregator
-
-#### Assets
-
-1. Unencrypted input shares.
-1. Input share decryption keys.
-1. Client identifying information.
-1. Aggregate shares.
-1. Aggregator identity.
-
-#### Capabilities and mitigations
-
-1. Aggregators may defeat the robustness of the system by emitting incorrect
-   aggregate shares.
-     * There is no way for aggregators to detect a fraudulent aggregate shares
-       except by applying heuristics to aggregate results that are outside of
-       DAP's scope. For instance it may be apparent from the aggregate result
-       that one or more aggregators have emitted an incorrect aggregate share.
-1. If Clients reveal identifying information to Aggregators (such as a trusted
-   identity during Client authentication), Aggregators can learn which Clients
-   are contributing reports.
-     1. Aggregators may reveal that a particular Client contributed reports.
-     1. Aggregators may attack robustness by selectively omitting reports from
-        certain Clients.
-          * For example, omitting submissions from a particular geographic
-            region to falsely suggest that a particular localization is not
-            being used.
-     * Exposing metadata to Aggregators can be mitigated by deploying an
-       anonymizing proxy (see {{anon-proxy}}).
-1. Individual Aggregators may compromise availability of the system by refusing
-   to emit aggregate shares.
-     * The DAP and VDAF threat model already assumes that robustness only holds
-       if both aggregators are honest, so a loss of availability is no worse.
-1. Violate robustness. Any Aggregator can collude with a malicious Client to
-   craft a proof that will fool honest Aggregators into accepting invalid
-   measurements.
-     * The VDAF threat model already assumes that robustness only holds if both
-       aggregators are honest.
-1. Aggregators (and the Collector) can count the total number of input shares,
-   which could compromise user privacy (and differential privacy {{dp}}) if the
-   presence or absence of a share for a given user is sensitive.
-   * Clients can ensure that aggregate counts are non-sensitive by generating
-     reports independently of user behavior (see {{network-attacker}}.
-   * Clients, especially in deployments that cannot schedule report uploads at a
-     fixed time (e.g., an application that does not run persistently) can also
-     apply local differential privacy to measurements before constructing
-     reports.
+DAP aims to achieve the privacy and robustness security goals defined in
+{{Section 9 of !VDAF}}, even in the presence of an active attacker. It is
+assumed that the attacker may control the network and have the ability to
+control a subset of of Clients, one of the Aggregators, and the Collector for a
```

Comment:
Please note the change here -- I'm not sure it's useful to define an adversary who can arbitrarily control all of the parties. For example, under the old scenario, the attacker knows the Leader's TLS private key as well as both aggregators' HPKE private keys (since it knows the "secret state of any party"), and is in control of the network, and as such can MitM the Client<->Leader connections then decrypt each measurement as it arrives.

We could call this out as another item outside of our threat model, but I think it makes more sense to restrict attacker control to at most one aggregator -- DAP & VDAF provide no guarantees if the aggregators collude.